src/main/java/uk/gov/hmcts/ccd/data/user/CachedUserRepository.java

```diff
@@ -46,7 +47,13 @@ public UserDefault getUserDefaultSettings(String userId) {
         return userRoles.computeIfAbsent("userRoles", e -> userRepository.getUserRoles());
     }
 
+    @Override
     public Set<SecurityClassification> getUserClassifications(String jurisdictionId) {
         return jurisdictionToUserClassifications.computeIfAbsent(jurisdictionId, userRepository::getUserClassifications);
     }
+
+    @Override
+    public SecurityClassification getHighestUserClassification() {
+        return userHighestSecurityClassification.computeIfAbsent("userHighestSecurityClassification", s -> userRepository.getHighestUserClassification());
```

Comment:
do we really need to cache this? can't we just forward directly to the user repo?